apps/frontend/app/pages/article/article.tsx

```diff
@@ -23,6 +23,9 @@ export function ArticlePage({
   const localizedHeadline = getLocalizedContent(article.article.headline);
   const localizedText = getLocalizedContent(article.article.text);
 
+  const onlySummary =
+    article.article.text["en"] === "" && article.article.text["de"] === "";
```

Comment:
can we make this more bulletproof please? because in the future we will receive null for text.

```suggestion
    !(article.article.text?.["en"] && article.article.text?.["de"]);
```

This should work if text is null, if de is null and if de = "", because an empty string is falsy in JS